src/components/HomePage/index.tsx

```diff
@@ -2,20 +2,18 @@ import * as React from "react";
 import { Query } from "react-apollo";
 import { Link } from "react-router-dom";
 
-import { Button, Carousel, Loader, ProductListItem } from "..";
+import { Button, Carousel, Loader, ProductListItem, ProductsFeatured } from "..";
 import { ProductsList } from "../../core/types/saleor";
-import { generateCategoryUrl, generateProductUrl } from "../../core/utils";
+import { generateCategoryUrl, maybe } from '../../core/utils';
```

Comment:
We use double quotes in the project. You can use Prettier to automatically format js code.